modules/oe_webtools_analytics/modules/oe_webtools_analytics_rules/src/Form/WebtoolsAnalyticsRuleForm.php

```diff
@@ -40,6 +40,13 @@ public function form(array $form, FormStateInterface $form_state): array {
       '#disabled' => !$rule->isNew(),
     ];
 
+    $form['multilingual'] = [
+      '#type' => 'checkbox',
```

Comment:
This form entry should be disabled/hidden if the website doesn't have the path module enabled. By doing so we ensure that the functionality is disabled completely in the super-rare case the module is not there.

Reply:
Done